dfc/proxy.go

```diff
@@ -837,6 +837,8 @@ func (p *proxyrunner) renamelocalbucket(bucketFrom, bucketTo string, lbmap4ren *
 	msg.Value = lbmap4ren
 	jsbytes, err := json.Marshal(msg)
 	assert(err == nil, err)
+	// only needs to tell target of the rename, proxies will get the sync call later
```

Comment:
Nitpick: consider moving these two lines right after `p.smap.copyL(smap4bcast)` so all the modifications to `smap4bcast` are in the same place.

Reply:
will do
thanks